flake.nix

```diff
@@ -62,8 +71,85 @@
               fourmolu
               haskell-language-server
               hlint
-            ]);
+            ])++ [ cardanoPkgs.cardano-node cardanoPkgs.cardano-cli ] ++ [pkgs.postgresql];
         });
+      testnet = perSystem (system:
+        let pkgs =nixpkgsFor system;
+        in
+          pkgs.runCommand "setPrivateNetwork" {buildInputs=(with pkgs; [git]);} (builtins.readFile ./setPrivateNetwork.sh)
+        );
+      runTestnet = self.testnet.x86_64-linux.testnet;
```

Comment:
I don't understand the purpose of this to be honest. Why limit yourself to `x86_64-linux`? The flake now works for various platforms, including ARM

Reply:
It was for fast access in my terminal, I would update a more clean nix file later.